spec/fixtures/config.yml

```diff
@@ -18,6 +18,8 @@ rollbar:
   proxy_port: '80'
 swift:
   auth_url: http://127.0.0.1:8080/auth/v1.0
+  auth_version: 'v1'
+  container: 'ERA'
```

Comment:
Does this need to be lower case?

Reply:
Not for 'v1' authentication which is what we are currently using.